server_test.go

```diff
@@ -1,15 +1,34 @@
 package main
 
 import (
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
 	"testing"
-
-	testsimplex "github.com/simplex-chat/test-simplex-server"
 )
 
+var server *string
+
+func TestMain(m *testing.M) {
+	server = flag.String("server", "http://localhost:8080", "simplex messaging server URI")
+	os.Exit(m.Run())
+}
+
 // TestHello tests "Hello Heroku"
 func TestHello(t *testing.T) {
-	ok, err := testsimplex.RunTestHello("http://localhost:8080")
-	if !ok {
+	fmt.Println(*server)
+	resp, err := http.Get(*server)
+	if err != nil {
 		t.Error(err)
 	}
+	defer resp.Body.Close()
+	body, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		t.Error(err)
+	}
+	if string(body) != "Hello World\n" {
+		t.Error(string(body))
+	}
```

Comment:
Proposing to switch to asserts, e.g. https://godoc.org/gotest.tools/assert

Reply:
possibly - I am evolving it - new PR coming.
I still didn't figure out what's the best way to write server tests.